### PR TITLE
Feature/add controllers question areas question sets and questions question sets

### PR DESCRIPTION
**Pull request by GagarinPavel**

### CODE REVIEW

File: AIS/AIS.API.Tests/Validators/QuestionAreasQuestionSets/ChangeQuestionAreasQuestionSetsValidatorTests.cs [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+namespace AIS.API.Tests.Validators.QuestionAreasQuestionSets
+{
+    public class ChangeQuestionAreasQuestionSetsValidatorTests
+    {
+        [Fact]
+        public void Validate_ValidModel_ReturnsTrue()
+        {
+            var validator = new ChangeQuestionAreasQuestionSetsViewModelValidator();
+            var model = new ChangeQuestionAreasQuestionSetsViewModel
```
Comment (Blackolf931) [inline AIS/AIS.API.Tests/Validators/QuestionAreasQuestionSets/ChangeQuestionAreasQuestionSetsValidatorTests.cs:14]: Use model from ValidQuestionsQuestionSetsViewModels.cs

File: AIS/AIS.API.Tests/Validators/QuestionAreasQuestionSets/ChangeQuestionAreasQuestionSetsValidatorTests.cs [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+            // Assert
+            result.IsValid.ShouldBeTrue();
+        }
+
+        [Fact]
+        public void Validate_QuestionSetIdLessThanOne_ReturnsFalse()
+        {
+            var validator = new ChangeQuestionAreasQuestionSetsViewModelValidator();
+            var model = new ChangeQuestionAreasQuestionSetsViewModel
```
Comment (Blackolf931) [inline AIS/AIS.API.Tests/Validators/QuestionAreasQuestionSets/ChangeQuestionAreasQuestionSetsValidatorTests.cs:31]: the same

File: AIS/AIS.API.Tests/Validators/QuestionAreasQuestionSets/ChangeQuestionAreasQuestionSetsValidatorTests.cs [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+            // Assert
+            result.Errors.ShouldNotBeEmpty();
+            result.Errors.ShouldContain(x => x.PropertyName == nameof(model.QuestionSetId));
+        }
+
+        [Fact]
+        public void Validate_QuestionAreaIdLessThanOne_ReturnsFalse()
+        {
+            var validator = new ChangeQuestionAreasQuestionSetsViewModelValidator();
+            var model = new ChangeQuestionAreasQuestionSetsViewModel
```
Comment (Blackolf931) [inline AIS/AIS.API.Tests/Validators/QuestionAreasQuestionSets/ChangeQuestionAreasQuestionSetsValidatorTests.cs:49]: the same

File: AIS/AIS.API.Tests/Validators/QuestionsQuestionSets/ChangeQuestionsQuestionSetsValidatorTests.cs [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+namespace AIS.API.Tests.Validators.QuestionsQuestionSets
+{
+    public class ChangeQuestionsQuestionSetsValidatorTests
+    {
+        [Fact]
+        public void Validate_ValidModel_ReturnsTrue()
+        {
+            var validator = new ChangeQuestionsQuestionSetsViewModelValidator();
+            var model = new ChangeQuestionsQuestionSetsViewModel
```
Comment (Blackolf931) [inline AIS/AIS.API.Tests/Validators/QuestionsQuestionSets/ChangeQuestionsQuestionSetsValidatorTests.cs:14]: the same

**Review by Blackolf931 — CHANGES_REQUESTED** [review]

**Review by hromau — APPROVED** [review]

**Review by Blackolf931 — APPROVED** [review]

**Review by stas1408123 — APPROVED** [review]

**Pull request merged**